components/popover.json

```diff
@@ -1,11 +1,21 @@
 {
-  "popover": {
-    "comment": "Applied to popovers",
-    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=5282%3A497",
-    "primary": {
-      "#normal": {
-        "background": "@theme-background-solid-primary-normal"
-      }
+    "popover": {
+        "comment": "Applied to popovers",
+        "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=5282%3A497",
+        "primary": {
+            "#normal": {
+                "background": "@theme-background-solid-primary-normal"
+            }
+        },
+        "audio-dump": {
+            "#normal": {
```

Comment:
audio dump is too specific in the popover component this should be added as secondary. Also this is already defined within modal which you could use